easybuild/easyconfigs/m/minieigen/minieigen-0.5.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'minieigen'
+version = '0.5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://launchpad.net/minieigen/'
+description = """A small wrapper for core parts of EIgen, c++ library for linear algebra."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [PYPI_SOURCE]
+
+dependencies = [
+    ('Boost', '1.60.0', versionsuffix),
+    ('Eigen', '3.2.7'),
+    ('Python', '2.7.11'),
+]
+
+sanity_check_paths = {
+    'files': ['lib/python2.7/site-packages/minieigen.%s' % SHLIB_EXT],
```

Comment:
please use `%(pyshortver)s` rather than hardcoding `2.7`

```python
    'files': ['lib/python%%(pyshortver)s/site-packages/minieigen.%s' % SHLIB_EXT],
```